Rakefile

```diff
@@ -979,8 +979,12 @@ namespace :release do
 
   desc "Perform a full build of the bundle and app"
   task :build => ['bundle:build', 'bundle:verify_linkage', 'bundle:test', 'app:build', PKG_DIR] do
-    output = `#{XCODEBUILD_COMMAND} -showBuildSettings | grep -w BUILT_PRODUCTS_DIR`.strip
-    build_dir = output.split('= ').last
+    build_dir = ""
+    Dir.chdir('app') do
+      output = `#{XCODEBUILD_COMMAND.join(" ")} -showBuildSettings | grep -w BUILT_PRODUCTS_DIR`.strip
+      build_dir = output.split('= ').last
+    end
+
```

Comment:
`XCODEBUILD_COMMAND` was once a string, is now an array. Used to assume it was being ran in the root dir too.